vizro-ai/docs/pages/user-guides/install.md

```diff
@@ -1,68 +1,167 @@
+
 
-More details can be found in the [environment setup guide](../user-guides/api-setup.md).
+??? example "How to override the default location of the .`env` file:"
 
-??? tip "Beginners/Code novices"
-    If you are a beginner or new to coding and wish to avoid using the terminal, you can follow these steps:
 
-    - Install [Anaconda Navigator](https://www.anaconda.com/download)
-    - Create a new environment as detailed [here](https://docs.anaconda.com/free/navigator/tutorials/manage-environments/), and select a Python version `>=3.8`
 
-## Install
+    ```py
+    from dotenv import load_dotenv, find_dotenv
+    from pathlib import Path
 
-To install Vizro-AI from the Python Package Index (PyPI), utilize [`pip`](https://pip.pypa.io/en/stable/) in your terminal with the following command:
+    # Specify the exact path to your .env file
+    env_file = Path.cwd() / ".env"  # Adjust the path as needed
+
+    # Alternatively, specify a different .env file name
+    env_file = find_dotenv(".env.dev")  # Replace ".env.dev" with your file name
+
+    # Load the specified .env file
+    load_dotenv(env_file)
+    ```
+    Refer to [python-dotenv documentation](https://saurabh-kumar.com/python-dotenv/reference/) for further information.
+
+!!! warning "Don't share your secret API key!"
+
+    You should avoid committing the `.env` file to version control. You can do this for Git by adding `.env` to your `.gitignore` file.
+
+
+__Method 2: Set an environment variable for all projects__
+
+To make the OpenAI API key available for all projects, you can set it as a system environment
+variable. Please refer to the section ["Set up your API key for all projects"](https://platform.openai.com/docs/quickstart/step-2-setup-your-api-key?context=python)
+in the OpenAI documentation. (It is under the dropdown of "Step 2: Set up your API key").
+
+The documentation provides step-by-step instructions for setting up the API key as an environment
+variable, on operating systems including Windows and MacOS.
+
+
+__Set the base URL (optional)__
+
+You might need to provide the base URL if you are using a custom OpenAI resource endpoint.
+
+The API base URL used for the OpenAI connector is set to `https://api.openai.com/v1` by default.
+If you are using a custom API endpoint, for example, if your organization has a designated API gateway,
+you can change the base URL by setting it as an environment variable.
+
+
+Follow the approach above in Method 2 to add the environment variable `OPENAI_API_BASE` for use by all projects.
+
+
+## Install Vizro
+
+To install Vizro-AI, use [`pip`](https://pip.pypa.io/en/stable/) in your terminal window:
 
 ```bash
 pip install vizro_ai
 ```
 
-While you can execute code from the tutorials and user guides using a Python script, using a Jupyter notebook is often considered more convenient. You can install `jupyter` with the following command:
+## Use Vizro-AI inside Anaconda Navigator
```

Comment:
This PR is still in draft so I'm not after full review right now, but I have a question please @antonymilne or @maxschulz-COL.

Do we anticipate that anyone will want to use Vizro-AI within Anaconda Navigator.

I would prefer to remove this section if not because we've asked them to set up access to a LLM, which requires some technical know-how, and it seems to me if they can do that, they're not likely to be upset about using a terminal, so won't want to use Navigator to avoid terminal usage.

I'm happy to retain it in Vizro but wondered about dropping this section from these docs.

WDYT?

Reply:
I agree! Let's remove that section!